apps/heft/src/plugins/FileGlobSpecifier.ts

```diff
@@ -110,9 +113,16 @@ export async function watchGlobAsync(
   const rawFiles: string[] = await glob(pattern, options);
 
   const results: Map<string, IWatchedFileState> = new Map();
-  for (const file of rawFiles) {
-    results.set(file, fs.getStateAndTrack(cwd ? path.resolve(cwd, file) : path.normalize(file)));
-  }
+  await Async.forEachAsync(
+    rawFiles,
+    async (file: string) => {
+      const state: IWatchedFileState = await fs.getStateAndTrackAsync(path.normalize(file));
```

Comment:
This is a regression. This is supposed to still check `absolute ? path.normalize(file) : path.resolve(cwd, file)`